urpmi: add pages (#12025)
tldr-pages/tldr: pages/linux/urpmf.md

# urpmf

> > Mageia's command for finding files in packages and querying information about them.

Reviewer: ```suggestion
> Find files in packages and query information about them in Mageia.
```

> > NOTE: Mageia documentation uses medium and repository as synonymous.

Reviewer: ```suggestion
```
 I don't think it is required to mention it here, as this command's page doesn't reference both of them.

# urpmf

> Mageia's command for finding files in packages and querying information about them.

> > NOTE: Mageia documentation uses media and repository as synonymous.

Reviewer 2: ```suggestion
> NOTE: Mageia documentation uses medium and repository as synonymous.
```

> See also: `urpmi`, `urpme`, `urpmi.addmedia`, `urpmi.removemedia`, `urpmi.update`, `urpmq`.
> More information: <https://wiki.mageia.org/en/URPMI#urpmi.removemedia>.

- Search for packages that contain a file:

`urpmf {{filename}}`

- Search for packages that contain both a keyword [a]nd another in their summaries:

`urpmf --summary {{keyword1}} -a {{keyword2}}`

- Search for packages that contain a keyword [o]r another in their descriptions:

`urpmf --description {{keyword1}} -o {{keyword2}}`

> - Search for packages that not contain a keyword in their name ignoring case distinction using "|" as the [F]ield separator (":" by default):

Reviewer 2: ```suggestion
- Search for packages that do not contain a keyword in their name ignoring case distinction using "|" as the [F]ield separator (":" by default):
```
